homeassistant/components/zwave_js/strings.sensor.json

```diff
@@ -0,0 +1,10 @@
+{
+  "state": {
+    "zwave_js__status": {
+      "wake up": "Wake up",
```

Comment:
We need to add a map between the zwave-js names and the state names we want to use in the integration, even if they are the same currently. If we add translations we need to be in control of the state names.